runtime/dotnet/core/Settings/BotSettings.cs

```diff
@@ -24,7 +24,7 @@ public class BotSettings
 
         public string Bot { get; set; }
 
-        public BotSkillSettings Skill { get; set; }
+        public BotSkillConfig SkillConfiguration { get; set; }
```

Comment:
I'd consider calling this either 1) `SkillConfiguration` or 2) `BotSkillConfiguration`.